### ISSUE TITLE
stopping gradient for encoder of Transformer

### ISSUE THREAD

**Issue by BaohaoLiao** [pytorch/fairseq#1053]
Hello,
I have a pretrained model. I want to load it to train for another task and don't want to train the encoder of transformer. Do you have any convenient way to do that?
